### ISSUE TITLE
RecycleRequiredException in Workflow service logs

### ISSUE THREAD

**Issue by marcusdroy** [boomerang-io/community#77]
**Describe the bug**
Witness the following error in workflow logs

org.apache.catalina.connector.CoyoteAdapter$RecycleRequiredException
	at org.apache.catalina.connector.CoyoteAdapter.checkRecycled(CoyoteAdapter.java:525)
	at org.apache.coyote.http11.Http11Processor.recycle(Http11Processor.java:1328)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.release(AbstractProtocol.java:1071)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:992)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1590)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:831)

**Comment by tlawrie** [comment]
This also occurs in the controller service for the lifecycle watcher threads

```[2021-03-04 01:24:48.026] [INFO] [46] net.bo.ku.se.KubeService                 : Starting Job Watcher #1 for Task (94588cd0-34ba-49a0-9fa8-4965626b5898)...
[2021-03-04 01:24:51.753] [INFO] [42] org.ap.ca.co.CoyoteAdapter               : Encountered a non-recycled response and recycled it forcedly.
org.apache.catalina.connector.CoyoteAdapter$RecycleRequiredException: null
	at org.apache.catalina.connector.CoyoteAdapter.checkRecycled(CoyoteAdapter.java:525) [tomcat-embed-core-9.0.22.jar!/:9.0.22]
	at org.apache.coyote.http11.Http11Processor.recycle(Http11Processor.java:1323) [tomcat-embed-core-9.0.22.jar!/:9.0.22]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.release(AbstractProtocol.java:1047) [tomcat-embed-core-9.0.22.jar!/:9.0.22]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:975) [tomcat-embed-core-9.0.22.jar!/:9.0.22]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1587) [tomcat-embed-core-9.0.22.jar!/:9.0.22]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-embed-core-9.0.22.jar!/:9.0.22]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-embed-core-9.0.22.jar!/:9.0.22]
	at java.lang.Thread.run(Thread.java:831) [?:?]
[2021-03-04 01:26:42.184] [WARN] [2551] com.sq.ok.OkHttpClient                   : A connection to https://10.0.0.1/ was leaked. Did you forget to close a response body?
[2021-03-04 01:26:42.184] [WARN] [2551] com.sq.ok.OkHttpClient                   : A connection to https://10.0.0.1/ was leaked. Did you forget to close a response body?
[2021-03-04 01:27:22.439] [INFO] [45] net.bo.ku.se.KubeService                 : Starting Job Watcher #7 for Task (94588cd0-34ba-49a0-9fa8-4965626b5898)...

**Comment by tlawrie** [comment]
Dropping priority due to related fix in boomerang-io/roadmap#169

**Comment by tlawrie** [comment]
Closing until it re-appears in Flow v4 with the entire service having been rewritten
